### PR TITLE
Clarified the service serving cert expiration time frame as per BZ161…

### PR DESCRIPTION
**Pull request by pneedle-rh**
Applies to branch/enterprise-3.11. Relates to https://bugzilla.redhat.com/show_bug.cgi?id=1618830.

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@openshift/team-documentation for peer review please. Please can someone also add 'Peer Review Needed', 'branch/enterprise-3.11' labels and 'Next Release' milestone?

File: dev_guide/secrets.adoc [hunk 1]
```diff
@@ -234,8 +234,8 @@ The certificate will be good for the internal service DNS name,
 `<service.name>.<service.namespace>.svc`.
 
 The certificate and key are in PEM format, stored in `tls.crt` and `tls.key`
-respectively. The certificate/key pair is automatically replaced when it gets
-close to expiration. View the expiration date in the
+respectively. The certificate/key pair is automatically replaced when expiry
```
Comment (kalexand-rh) [inline dev_guide/secrets.adoc:237]: s/expiry/expiration

Expiration is used far more often in the collection.

**Review by kalexand-rh — COMMENTED** [review]
I have one request, then it LGTM.

**Comment by pneedle-rh** [comment]
@barleyer I have updated the docs to reflect 'Service Serving Certificate Secrets' certificate/key pair expiration is within one hour. Can you please review this from a QE perspective?

**Comment by pneedle-rh** [comment]
@openshift/team-documentation QE ACK granted and SME discussion complete. Can someone please merge and CP for enterprise-3.11? Thanks!

**Pull request merged**